ArjunaJTS/jts/classes/com/arjuna/ats/internal/jts/orbspecific/ibmorb/recoverycoordinators/JavaIdlRCServiceInit.java

```diff
@@ -247,7 +247,7 @@ private static String initRCKey() {
     }
 
     private static final String POA_NAME_PREFIX = "RcvCo-";
-    private static String OBJ_KEY_PREFIX = ""; // is there an equivalent of JacORBs jacorb.implname
+    private static String OBJ_KEY_PREFIX = ""; // Prefix for the Recovery Coordinator's key (i.e. RC_KEY)
```

Comment:
Thank you, I couldn't spot what the RC_KEY is used for, are you able to say?

Reply:
It is very likely that the prefix defined in `OBJ_KEY_PREFIX` is used for logging purposes (in line with `jacorb.implname`)

Reply:
I didn't find it logged, but I did find it https://github.com/jbosstm/narayana/blob/c5f02d07edb34964b64341974ab689ea44536603/ArjunaJTS/jts/classes/com/arjuna/ats/internal/jts/orbspecific/javaidl/recoverycoordinators/RecoverIOR.java#L49 - but I don't see things putting it in IOR - @mmusgrov do you know about this area?

Reply:
Do you think we can go ahead with this PR and then add further explanations in case @mmusgrov gives us additional info?

Reply:
If `OBJ_KEY_PREFIX`'s purpose was to emulate `jacorb.implname` then `OBJ_KEY_PREFIX` will create an unique reference for the recovery coordinator (as it is part of the `RC_KEY`) across different processes. Problem is that I do not understand how `OBJ_KEY_PREFIX` can change across different processes as it is hard-coded (on the other hand `jacorb.implname` is a property). That is why the only purpose I can see for `OBJ_KEY_PREFIX` is to have a well-known prefix in the logs. Maybe, the purpose of `OBJ_KEY_PREFIX` is to give a possibility to put a prefix to better identify different recovery managers for different ORBs (e.g. the RecoverIOR object of ibmorb and openjdkorb)

Reply:
Please can you raise a JBTM to investigate it? Once raised, I don't think it needs to block this PR any further

Reply:
Please see below a comment that is being removed related to jacorb.implname says "persistent POAs can't be anonymous, need a name". I am not sure if it's related but to this usage but anyway noting it here too.